checkov/bicep/graph_builder/context_definitions.py

```diff
@@ -0,0 +1,66 @@
+from pathlib import Path
+from typing import cast, List, Tuple, Dict, Any
+
+from checkov.common.util.suppression import collect_suppressions_for_report
+
+from pycep.typing import BicepJson
+
+
+BICEP_COMMENT = "//"
+DEFINITIONS_KEYS_TO_PARSE = {"parameters": "parameters", "resources": "resources"}
+
+
+def build_definitions_context(
```

Comment:
Nice implementation! 
It's a very long method is it possible to extract some of the code into other functions?